docs/readthedocs/source/doc/UserGuide/python.md

```diff
@@ -84,9 +84,13 @@ You may test if the installation is successful using the interactive Python shel
 * Try to run the example code below to verify the installation:
 
   ```python
-  from bigdl.orca import init_orca_context
+  from bigdl.dllib.nncontext import init_nncontext
```

Comment:
why change from orca to dllib?

Reply:
Kai said it's a little confused init orca context, but call dllib's library.

Reply:
But still why do we use dllib instead of orca to verify?

Reply:
What Orca code do you suggest to verify? Seems it is not easy to find some simplest code in Orca?

Reply:
`init_orca_context` on local? if this works, the installation should be successful?

Reply:
So do not modify this? Already there: https://bigdl.readthedocs.io/en/latest/doc/UserGuide/python.html#interactive-shell

Reply:
Maybe show what the example output should look like if the installation is successful